lib/web/scripts/desktop/configure-ad.ps1

```diff
@@ -136,6 +136,7 @@ $CA_CERT_YAML = $CA_CERT_PEM | ForEach-Object { "        " + $_  } | Out-String
 
 $NET_BIOS_NAME = (Get-ADDomain).NetBIOSName
 $LDAP_USERNAME = "$NET_BIOS_NAME\$SAM_ACCOUNT_NAME"
+$LDAP_USER_SID=(Get-ADUser -Identity svc-teleport).SID.Value
```

Comment:
Shouldn't we use `$SAM_ACCOUNT_NAME` here instead of hardcoding the value?

```suggestion
$LDAP_USER_SID=(Get-ADUser -Identity $SAM_ACCOUNT_NAME).SID.Value
```

Reply:
Yes, copy-paste error on my part.